src/core/electrostatics_magnetostatics/coulomb.hpp

```diff
@@ -0,0 +1,51 @@
+#ifndef ESPRESSO_COULOMB_SWITCH_HPP
+#define ESPRESSO_COULOMB_SWITCH_HPP
+
+#include "statistics.hpp" // Observable_stat
+
+#ifdef ELECTROSTATICS
+
```

Comment:
Can you please put everything here into a namespace `Coulomb`, and shorten the names e.g. `pressure_inline_add_pair_pressure` -> `add_pair_pressure` and so on. The same for dipolar.